src/docs/markdown/caddyfile/directives/log.md

```diff
@@ -17,7 +17,7 @@ log {
 ```
 
 - **output** configures a where to write the logs to. See [Output modules](#output-modules) below. Default: `stderr`
-- **format** describes how to encode, or format, the logs. See [Format modules](#format-modules) below. Default `console`
+- **format** describes how to encode, or format, the logs. See [Format modules](#format-modules) below. Default: `console` if terminal, otherwise `json`
```

Comment:
```suggestion
- **format** describes how to encode, or format, the logs. See [Format modules](#format-modules) below. Default: `console` if `stdout` is detected to be a terminal, `json` otherwise.
```
Just some slight wording change, plus a bit more specific about the actual behaviour (looks at `stdout` to determine if it's a terminal).

Reply:
great suggestion! i just amended my previous commit.